homer.root/usr/share/clearwater/cassandra-schemas/homer.sh

```diff
@@ -16,7 +17,24 @@ then
     /usr/share/clearwater/bin/poll_cassandra.sh --no-grace-period
   done
 
-  echo "CREATE KEYSPACE homer WITH REPLICATION = {'class': 'SimpleStrategy', 'replication_factor': 2};
+  replication_str="{'class': 'SimpleStrategy', 'replication_factor': 2}"
+
+  # If local_site_name and remote_site_names are set then this is a GR
```

Comment:
Should this be in a common Bash file that homestead_provisioning.sh, homer_provisioning.sh etc. just dot in? Makes it easier to change this in future.